src/index.js

```diff
@@ -69,7 +69,8 @@ function renderDrama(message, share, sharePath, teaser) {
         <span id="fight"></span>
         <h6>
             <a id="more" href="/">Generate more drama!</a> (or press enter)
-            <br /><br />
+            <br />
+            <br />
             This website is made in jest - don't take it too seriously!
             <br />
             Developed by md678685; PRs welcome on <a href="https://github.com/md678685/spigot-drama-generator">GitHub</a>.
```

Comment:
Since it's in the scope
```suggestion
            Developed by mdcfe; PRs welcome on <a href="https://github.com/mdcfe/spigot-drama-generator">GitHub</a>.
```